exchanges/bitfinex/bitfinex_test.go

```diff
@@ -84,9 +102,9 @@ func TestGetAccountInfoV2(t *testing.T) {
 
 func TestGetV2FundingInfo(t *testing.T) {
 	if !areTestAPIKeysSet() {
-		t.SkipNow()
+		t.Skip("api keys are not set or invalid")
 	}
-	_, err := b.GetV2FundingInfo("fUSD")
+	_, err := b.GetV2FundingInfo("fUSD,fUST")
```

Comment:
This test currently fails with the additional symbol but works fine with just one. Is there anything extra you did to get it working for you?

Reply:
Seems to me like it was just a mistake on my part putting multiple currencies there at once. And it only returns an individual